roles/ceph-config/tasks/main.yml

```diff
@@ -142,10 +142,4 @@
       mode: "0644"
       config_overrides: "{{ ceph_conf_overrides }}"
       config_type: ini
-    notify:
-      - restart ceph mons
-      - restart ceph osds
-      - restart ceph mdss
-      - restart ceph rgws
-      - restart ceph mgrs
-      - restart ceph rbdmirrors
+    notify: "{{ ceph_notify | default(omit) }}"
```

Comment:
not sure `omit` can work in a `notify` ?

Reply:
according to the ci it seems to work

Reply:
héhé `omit` FTW :1st_place_medal: